cpp/arcticdb/version/version_core.cpp

```diff
@@ -596,7 +596,7 @@ void set_output_descriptors(
 
 std::shared_ptr<std::unordered_set<std::string>> columns_to_decode(const std::shared_ptr<PipelineContext>& pipeline_context) {
     std::shared_ptr<std::unordered_set<std::string>> res;
-    ARCTICDB_DEBUG(log::version(), "Creating columns list with {} bits set", pipeline_context->overall_column_bitset_->count());
+    ARCTICDB_DEBUG(log::version(), "Creating columns list with {} bits set", pipeline_context->overall_column_bitset_ ? pipeline_context->overall_column_bitset_->count() : -1);
```

Comment:
Is this meant to be part of this change?

Reply:
Yes, tests were crashing because `pipeline_context->overall_column_bitset_` is optional and in some cases it was `nullopts`